res/verification_template_dinsic.eml

```diff
@@ -98,14 +100,16 @@ pre, code {
                             <td>
                             </td>
                             <td class="logo">
-                                    <img src="https://matrix.i.tchap.gouv.fr/images/tchap%%20144x144.png" width="64" height="64" alt="Tchap"/>
+                                    <img src="https://matrix.i.tchap.gouv.fr/images/tchap192.png" width="192" height="192" alt="Tchap"/>

```

Comment:
I would prefer render to a 64x64 logo
Can we use the tchap192.png with width="64" height="64"